_includes/components/site_nav.html

```diff
@@ -1,25 +1,23 @@
 {%- comment -%}
-  Most of this is the taken directly from just-the-docs on github.
-  I just changed the nav structure a bit, to include the table of contents.
-
-  Include as: {%- include_cached components/site_nav.html -%}
+  Include as: {%- include_cached components/site_nav.html all=bool -%}
   Depends on: site.
-  Results in: HTML for the site-nav.
+  Results in: cached HTML for the main navigation when `all` is nil or false;
+    includes links to pages excluded from the main navigation when `all` is true.
   Includes:
-    components/nav.html
+    components/nav/pages.html
   Overwrites:
     pages_top_size, collections_size, collection_entry,
     collection_key, collection_value, collection.
 {%- endcomment -%}
 
-<nav>
+<nav aria-label="Main" id="site-nav" class="site-nav">
```

Comment:
I'm pretty sure that change is the reason I had this file duplicated. I'll put it back with a comment explaining this.